packages/components/src/components/drawer/drawer.scss

```
@@ -80,6 +80,7 @@ $spacings: (
 
 	.db-drawer-header {
 		// We need this variable to overwrite it inside the header
+		/* stylelint-disable-next-line db-ui/use-spacings */
```

```suggestion
		/* TODO: We would need to correct this reference in the future */
		/* stylelint-disable-next-line db-ui/use-spacings */
```

this is a potential TODO, isn't it ?

---

Nope. We need this css-property to change the padding based on desktop and mobile. Our own plugin doesn't work perfectly in our repository because we need the flexibility inside a component. A product wouldn't need this flexibility in 90% of the time.

---

agreed.